From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
From: Bjarne Koll <lynxplay101@gmail.com>
Date: Wed, 8 Sep 2021 20:52:25 +0200
Subject: [PATCH] Gradle project changes

This commit modifies the gradle build configuration of the server
api to fit the ktp fork.

diff --git a/build.gradle.kts b/build.gradle.kts
index 689d470ebe2c7e25e4c87f6127978d1d283e55b5..231711179ac2c75e561facb0899a453bf9d6f01c 100644
--- a/build.gradle.kts
+++ b/build.gradle.kts
@@ -98,3 +98,15 @@ tasks.withType<Javadoc>().configureEach {
         // Paper end
     )
 }
+
+// KTP start - add publishing
+publishing {
+    repositories {
+        maven {
+            name = "knockturnPublic"
+            url = uri("https://repo.knockturnmc.com/content/repositories/knockturn-public/")
+            credentials(PasswordCredentials::class)
+        }
+    }
+}
+// KTP end
\ No newline at end of file
